crates/crash-handler/src/lib.rs

```diff
@@ -44,6 +45,11 @@ fn handle_panic(panic_info: &PanicInfo<'_>) {
     // Wait till the logs have been flushed
     aptos_logger::flush();
 
+    // Do not kill the process if the panics happened at move-bytecode-verifier.
```

Comment:
Nit: could we add a TODO here that says once `can_unwind` is stable, we should assert it? Just to be sure we aren't ignoring something that cannot unwind? https://github.com/rust-lang/rust/issues/92988